From b19b164aa0ecff48824a21976b71f954dcff527b Mon Sep 17 00:00:00 2001
From: Pierre Neidhardt <ambrevar@gmail.com>
Date: Sat, 11 Apr 2015 10:27:16 +0200
Subject: [PATCH] Makefile: Fix man page installation path

Latest FHS replaces /usr/man by /usr/share/man.

If folder `man1` does not exist, `$(INSTALL) metapixel.1 $(MANDIR)/man1` will
install the man page as `man1`, hence the -D flag to force folder creation.
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index e40e0e1..dc1ccaf 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@ INSTALL = install
 MANPAGE_XSL = /usr/share/xml/docbook/stylesheet/nwalsh/current/manpages/docbook.xsl
 
 BINDIR = $(PREFIX)/bin
-MANDIR = $(PREFIX)/man
+MANDIR = $(PREFIX)/share/man
 
 VERSION = 1.1.0
 
@@ -66,7 +66,7 @@ install : metapixel metapixel.1
 	$(INSTALL) -d $(BINDIR)
 	$(INSTALL) metapixel $(BINDIR)
 	$(INSTALL) metapixel-prepare $(BINDIR)
-	$(INSTALL) metapixel.1 $(MANDIR)/man1
+	$(INSTALL) -D metapixel.1 $(MANDIR)/man1/metapixel.1
 #	$(INSTALL) imagesize $(BINDIR)
 #	$(INSTALL) sizesort $(BINDIR)
 
